index.d.ts

```diff
@@ -0,0 +1,9 @@
+import type { MeterProvider } from "@opentelemetry/api-metrics";
+
+declare module "opentelemetry-node-metrics" {
```

Comment:
I hope i did this right. I have never written a `d.ts` file before.

Reply:
wouldn't be jsdoc annotations already good enough for this simple use case?